packages/core/lib/services/credential/service.js

```diff
@@ -84,6 +89,28 @@ class CredentialService {
     const {credential} = params;
     return verifyCredential(credential, {resolver, revocationApi: {dock}});
   }
+
+  filterCredentials(params) {
+    const {credentials, presentationDefinition, holderDid} = params;
+    const result = pex.selectFrom(
+      presentationDefinition,
+      credentials,
+      holderDid,
+    );
+
+    return result;
+  }
+
+  evaluatePresentation(params) {
+    const {presentation, presentationDefinition} = params;
```

Comment:
do we need to worry about validating these params before passing them to pex?

same with function above

Reply:
and what about tests?

Reply:
pex will validate that for us